8vim/src/main/res/values/custom_colors.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <array name="custom_colors">
+        <item>#F44336</item>
+        <item>#FF5722</item>
+        <item>#E91E63</item>
+        <item>#FFFFFF</item>
+        <item>#FFFFFF</item>
+        <item>#673AB7</item>
+        <item>#2196F3</item>
+        <item>#03A9F4</item>
+        <item>#4CAF50</item>
+        <item>#8BC34A</item>
+        <item>#FFEB3B</item>
+        <item>#FFC107</item>
+        <item>#FF9800</item>
+        <item>#000000</item>
+    </array>
+</resources>
```

Comment:
This seems pretty pointless. The colors on the color seekbar should be self generated **starting** from white and **ending** in black.